163204 Create Solution Template Blades correctly

ashutosh-kumar1 opened the pull request:
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)


- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* createUiDefinition.json file added as a Solution Template GUI
* mainTemplate.json file added to be used for Solution Template

Commit:
163204 Create Solution Template Blades correctly

Commit:
163204 Updating file to resolve validation errors

robe070 reviewed (commented):
I see you removed the `visible` condition. I did not see an error for this in the pipeline. So was it reported as an error or were you just simplifying the template in order that it would be more likely to pass validation?

And, you can run the validation test yourself before submitting to the pipeline. Refer here: https://github.com/robe070/cookbooks/wiki/Lansa-Templates---Validation-policy-using-ARM-TTK

ashutosh-kumar1 commented:
@robe070 , I had removed the visible condition because the database type must be visible to the user. Moreover, I had added few parameters in the output that caused the error to occur.
I am locally testing it locally with the arm-ttk, therefore it is taking me some time to commit. Will commit it after resolving all the errors.

Commit:
163204 Validated createUiDefinition file using arm-ttk
